Makefile

```diff
@@ -38,7 +38,7 @@ verify-dependency-bounds-sys: test-set-lower-bounds
 	RUSTFLAGS="${DENY}" cargo build ${MANIFEST_PATH_ARGS} --all-features
 
 clippy:
-	RUSTFLAGS="${DENY}" cargo clippy --all-targets --all-features -- -D clippy::needless_borrow -A clippy::upper-case-acronyms -A clippy::from_over_into
+	RUSTFLAGS="${DENY}" cargo clippy --all-targets --all-features -- -D clippy::cargo -D clippy::needless_borrow -A clippy::upper-case-acronyms -A clippy::from_over_into
```

Comment:
I'm guessing that you can do

...```--- -D clippy::all -D clippy::cargo``` with out hurting anything.

Reply:
We do have to explicitly allow ```from_over_into``` but ti looks like in exactly one place, so it makes more sense to put the allow in that place. As far as I can tell, the other explcit lints, either allow or deny, have no effect. So we might as well simplify a bit more here.